### ISSUE TITLE
Ambiguous meaning of warning : "Are you sure that your want to replace it?" <Yes|No>

### ISSUE THREAD

**Issue by hoonseo0409** [lra/mackup#1662]
This is related to: https://github.com/lra/mackup/issues/283#issue-36587590
Let's suppose you have a setting file in the storage(e.g. icloud): backup,
and you have a same setting file (with the different contents) in the disk: local.

If you run "mackup restore", because of this conflict, mackup should decide which setting file to use among the backup and loca, for example,

A file named /Users/Ryan/Dropbox/Mackup/Library/Preferences/com.irradiatedsoftware.SizeUp.plist already exists in the backup.
Are you sure that your want to replace **it** ? <Yes|No>

Then it is not very clear to me what "it" indicates, is it backup or local? From the context, I guess "it" indicates the backup, so if I choose "Yes", then the backup in the remote storage will be deleted and my local setting will be uploaded. But I found an article "https://ostechnix.com/backup-and-restore-application-settings-on-newly-installed-linux-system/" stating that "Mackup will ask you to overwrite the existing file with the backup files. Type Yes and hit ENTER to restore all configurations files.", which means "Yes means deleting the local setting and use the backup setting". This is confusing, so I think "it" in the warning message should be replaced with "backup" or "local" to make this clear.

In short, does "Yes" mean that the backup will be deleted and local will be uploaded?

**Comment by hoonseo0409** [comment]
I just found that the message is now "Are you sure that your want to replace it **with backup**?", which means if I answer Yes then the "backup setting" will be used and "local setting" will be moved and ignored. Let me know if I'm wrong.

**Comment by lra** [comment]
I don't understand.
If you have a file named "a" in Dropboxy and you run `mackup restore`, mackup will try to copy this file `a` into your home. That's it. Mackup has no database to remember if you renamed a file and `a` is now `b` in your home.

**Comment by vfonic** [comment]
@hoonseo0409 I believe "it" depends on which of the two commands you run: `mackup backup` or `mackup restore`.

Try to think what these commands do: `mackup backup` copies from local to Dropbox. So, _I assume_ it will overwrite anything you might have in Dropbox. Similarly `mackup restore` goes the other way around: Dropbox => local. It will overwrite local files.
